packages/app-store/_utils/nodeScheduler.ts

```diff
@@ -0,0 +1,296 @@
+      const bookingsWithScheduledJobs = await prisma.booking.findMany({
+        where: {
+          ...where,
+          scheduledJobs: {
+            isEmpty: false,
+          },
+        },
+      });
+      for (const booking of bookingsWithScheduledJobs) {
+        const updatedScheduledJobs = booking.scheduledJobs.filter(
+          (scheduledJob) => scheduledJob !== `${appId}_${webhook.id}`
+        );
+        await prisma.booking.update({
+          where: {
+            id: booking.id,
+          },
+          data: {
+            scheduledJobs: updatedScheduledJobs,
+          },
+        });
+      }
+    }
+
+    const deleteWebhook = await prisma.webhook.delete({
+      where: {
+        id: webhookId,
+      },
+    });
+    if (!deleteWebhook) {
+      throw new Error(`Unable to delete webhook ${webhookId}`);
+    }
+    return deleteWebhook;
+  } catch (err) {
+    return console.error(
+      `Error deleting subscription for user ${appApiKey.userId}, webhookId ${webhookId}, appId ${appId}`
+    );
+  }
+}
+export async function listBookings(appApiKey: ApiKey) {
+  try {
+    const where: Prisma.BookingWhereInput = {};
+    if (appApiKey.teamId) {
+      where.eventType = {
+        OR: [{ teamId: appApiKey.teamId }, { parent: { teamId: appApiKey.teamId } }],
+      };
+    } else {
+      where.userId = appApiKey.userId;
+    }
+    const bookings = await prisma.booking.findMany({
+      take: 3,
+      where: where,
+      orderBy: {
+        id: "desc",
+      },
+      select: {
+        title: true,
+        description: true,
+        customInputs: true,
+        responses: true,
+        startTime: true,
+        endTime: true,
+        location: true,
+        cancellationReason: true,
+        status: true,
+        user: {
+          select: {
+            username: true,
+            name: true,
+            email: true,
+            timeZone: true,
+            locale: true,
+          },
+        },
+        eventType: {
+          select: {
+            title: true,
+            description: true,
+            requiresConfirmation: true,
+            price: true,
+            currency: true,
+            length: true,
+            bookingFields: true,
+            team: true,
+          },
+        },
+        attendees: {
+          select: {
+            name: true,
+            email: true,
+            timeZone: true,
+          },
+        },
+      },
+    });
+    if (bookings.length === 0) {
+      return [];
+    }
+    const t = await getTranslation(bookings[0].user?.locale ?? "en", "common");
+
+    const updatedBookings = bookings.map((booking) => {
+      return {
+        ...booking,
+        ...getCalEventResponses({
+          bookingFields: booking.eventType?.bookingFields ?? null,
+          booking,
+        }),
+        location: getHumanReadableLocationValue(booking.location || "", t),
+      };
+    });
+
+    return updatedBookings;
+  } catch (err) {
+    return console.error(
```

Comment:
nit: I think it might be old code but it's better to use logger. We can go for `logger.error` here

Reply:
made edits. pls check child logger label.